mailqueue/utils.py

```diff
@@ -27,4 +27,4 @@ def upload_to(instance, filename):
     filename = filename.split('/')[-1]
     # Because instead of filesystem, email message
     # can have multiple attachments with the same filename
-    return 'mailqueue-attahcments/{0}_{1}'.format(get_random_string(length=24), filename)
+    return 'mailqueue-attachments/{0}_{1}'.format(get_random_string(length=24), filename)
```

Comment:
Nay, typos should be fixed some day.
The sooner the better.

Yet I propose to make this path configurable so ones who want old path can just configure it and default value should be typoless.